dac161s997.h

```diff
@@ -41,8 +42,11 @@
 
 /* Typedefs *******************************************************************/
 typedef enum {
-    DAC161S997_ALARM_LOW,   /**< Set output to undercurrent */
-    DAC161S997_ALARM_HIGH,  /**< Set output to overcurrent */
+    DAC161S997_ALARM_LOW_FAIL   = DAC161S997_ALARM_LOW_FAIL_ERR ,
+    DAC161S997_ALARM_LOW_SAT    = DAC161S997_ALARM_LOW_SAT_ERR ,
+    DAC161S997_ALARM_UNINIT     = DAC161S997_ALARM_UNINIT_ERR ,
+    DAC161S997_ALARM_HIGH_SAT   = DAC161S997_ALARM_HI_SAT_ERR,
+    DAC161S997_ALARM_HIGH_FAIL  = DAC161S997_ALARM_HI_FAIL_ERR ,
```

Comment:
I think documentation can be added here as well, just replace `Error flag` with `Alarm`